packages/connect-core/src/entities/Organization.ts

```diff
@@ -48,6 +48,10 @@ export default class Organization {
     return this.#connector.appsForOrg!(this.#address)
   }
 
```

Comment:
This is how an entity implements a subscription function. It's basically a mapping to the connector's corresponding function, much like it already was with single-call functions.